aclk/tests/install-fake-charts.d.sh.in

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+TARGET="@pluginsdir_POST@"
+BASE="$(cd "$(dirname "$0")" && pwd)"
+
+cp "$BASE/fake-charts.d.plugin" "$TARGET/charts.d.plugin"
```

Comment:
The file is `fake_charts.d.plugin` (not with a dash)

Reply:
Moved it.